content/docs/sidebar.json

```diff
@@ -131,7 +131,8 @@
           "stop-tracking-data",
           "update-tracked-data",
           "add-deps-or-outs-to-a-stage",
-          "merge-conflicts"
+          "merge-conflicts",
+          "share-a-dvc-cache"
```

Comment:
This ideally should be part of something else. Some ideas:

UG -> Data Management -> Shared Cache


here is in the How to, we can have - "How to avoid duplication of data in the team", "How to organize team storage for large raw datasets", etc, etc.

Reply:
> should be part of something else... UG -> Data Management -> Shared Cache

Agree . Didn't want to overcomplicate this PR. Should we go for a how-to for now and make an issue about the Data Mgmt section (would also lead to #520) instead?

Reply:
I'm fine with that. Let's create a ticket though to address it later.

Reply:
> Should we go for a how-to

I meant top-level guide. I agree that the topic is a bit broad for a how.

Reply:
Done: #2487

Reply:
👍